### PR TITLE
klavaro: 3.08 -> 3.09, patch format-string, espeak support

### PR DESCRIPTION
**Pull request by dtzWill**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/games/klavaro/default.nix [hunk 1]
```diff
@@ -1,17 +1,38 @@
...
   src = fetchurl {
     url = "mirror://sourceforge/klavaro/${name}.tar.bz2";
-    sha256 = "0qmvr6d8wshwp0xvk5wbig4vlzxzcxrakhyhd32v8v3s18nhqsrc";
+    sha256 = "12gml7h45b1w9s318h0d5wxw92h7pgajn2kh57j0ak9saq0yb0wr";
   };
 
   nativeBuildInputs = [ intltool makeWrapper pkgconfig ];
   buildInputs = [ curl gtk3 ];
 
+  patches = [ (builtins.toFile "format-string" ''
```
Comment (worldofpeace) [inline pkgs/games/klavaro/default.nix]: don't you think this is a rather, interesting way to patch things?
Is this your patch @dtzWill?

**Review by worldofpeace — COMMENTED** [review]
Verified everything executes and inspected that the `patchPhase` is doing things properly. (ran manually).

File: pkgs/games/klavaro/default.nix (hunk 1 above)
Comment (dtzWill) [reply pkgs/games/klavaro/default.nix]: Yes, this is mine :).  I like having small patch inline but can of course move to a separate file if preferred?

**Comment by dtzWill** [comment]
Closing, will submit espeak fixup in separate PR (and I think bump as well, actually) :).

**Pull request closed without merging**